node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -798,4 +822,18 @@ object NodeRuntime {
     val propose               = ProposeGrpcServiceV1.instance(blockApiLock)
     APIServers(repl, propose, deploy)
   }
+
+  @SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
+  private def runRdmbsMigrations[F[_]: Sync](serverDataDir: Path): F[Unit] =
```

Comment:
I am philosophically opposed to running db migrations as part of a normal program run. Not only this makes things harder to test in isolation, but also there is a risk of things going wrong on production. IMO it should be a conscious decision

Reply:
How do you feel about keeping this as a CLI option? In other words, you will have to run the node with `--migrate-db` to run migrations automatically.

Reply:
I think it's better. For what it's worth it might as well be a separate program